cartoframes/viz/style.py

```diff
@@ -60,33 +60,36 @@ def _init_style(self, data):
         else:
             raise ValueError('`style` must be a string or a dictionary')
 
-    def compute_viz(self, geom_type, variables={}):
+    def compute_viz(self, geom_type, variables={}, properties={}):
         style = self._style
         default_style = defaults.STYLE[geom_type]
+
         if isinstance(style, dict):
             if geom_type in style:
                 style = style.get(geom_type)
-            return self._parse_style_dict(style, default_style, variables)
+            return self._parse_style_dict(style, default_style, variables, properties)
         elif isinstance(style, str):
-            return self._parse_style_str(style, default_style, variables)
+            return self._parse_style_str(style, default_style, variables, properties)
         else:
             raise ValueError('`style` must be a string or a dictionary')
 
-    def _parse_style_dict(self, style, default_style, ext_vars):
+    def _parse_style_dict(self, style, default_style, ext_vars, ext_props):
         variables = merge_dicts(style.get('vars', {}), ext_vars)
         properties = merge_dicts(default_style, style)
+        allproperties = merge_dicts(properties, ext_props)
```

Comment:
This can overwrite the current style properties. We need to check if we want to do that or a deeper merge (for example, filterA & filterB).

Anyway, it should be nice to add tests to cover the `prop` case and the case when there is already a filter in the style.